### PR TITLE
fix: Update nvidia metric name

### PR DESCRIPTION
**Pull request by gracedo**
**What type of PR is this?**

bug

**What this PR does/ why we need it**:

nvidia grafana dashboard was using an outdated metric name

**Which issue(s) this PR fixes**:

https://jira.d2iq.com/browse/COPS-7132

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Checklist**

* [ ] *If a chart is changed, the chart version is correctly incremented.*
* [ ] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

**Comment by gracedo** [comment]
@tillt I updated the metric name but I'm not sure how to test nvidia, if you don't mind testing this fix thanks!

File: staging/nvidia/charts/grafana/dashboards/dcgm-exporter-dashboard.json [hunk 1]
```diff
@@ -100,7 +100,7 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "DCGM_FI_DEV_GPU_TEMP{instance=~\"${instance}\", gpu=~\"${gpu}\"}",
+          "expr": "DCGM_FI_DEV_DEC_UTIL{instance=~\"${instance}\", gpu=~\"${gpu}\"}",
```
Comment (tillt) [inline staging/nvidia/charts/grafana/dashboards/dcgm-exporter-dashboard.json:103]: That looks wrong - we are replacing the GPU device temperature with decoder utilization here.

**Review by tillt — CHANGES_REQUESTED** [review]
This smells like a red herring. We ought to look closer and see how the customer got to this conclusion.

File: staging/nvidia/charts/grafana/dashboards/dcgm-exporter-dashboard.json (hunk 1 above)
Comment (gracedo) [reply staging/nvidia/charts/grafana/dashboards/dcgm-exporter-dashboard.json:103]: oops sorry, I replaced the wrong one, should be replacing `DCGM_FI_DEV_GPU_UTIL` but perhaps it's still not the right metric..

**Pull request closed without merging**